### ISSUE TITLE
Price norm_method

### ISSUE THREAD

**Issue by jpmediadev** [ZhengyaoJiang/PGPortfolio#43]
Greetings!

Thank you for posting your incredible work!

if I understood you correctly - then the normalization of prices in the current code is not used
"norm_method": "relative", / "absolute"
I did not find where the function "pricenorm3d" is called 
the percentage of price increment/decrements is used as a result - y
Is it so that the "RAW" price values are used  - X?

How does the network summarize price data for the past periods when prices differed by an order of magnitude?
or is it done intentionally to take only the closest experience when prices were on the same scale?

I apologize if this is an obvious question, and also for my English

thank you

**Comment by DexHunter** [comment]
Did you check out the paper?

Quoting myself from [a previous issue](https://github.com/ZhengyaoJiang/PGPortfolio/issues/22#issuecomment-352323303)

`y` is price relative vector for two adjacent price vector(`v_t` & `v_{t-1}`) while Eq. (18) states the normalization of the price in the training set.  You can see the normalization code [for y](https://github.com/ZhengyaoJiang/PGPortfolio/blob/master/pgportfolio/marketdata/datamatrices.py#L168-L169) and [for X](https://github.com/ZhengyaoJiang/nntrader/blob/dev/nntrader/learn/network.py#L39)

Maybe this will helps a bit.

**Comment by jpmediadev** [comment]
thanks,

normalization code for y  - ok
but for X - 404

**Comment by DexHunter** [comment]
> but for X - 404

for X it's done at batch training, in code
```python
network = network / network[:, :, -1, 0, None, None]
```

So to put in simple English words, it's the batch of relative prices divided by the last period of the relative price.

**Comment by jpmediadev** [comment]
understood thanks!
